recipe/no_async.patch

```diff
@@ -0,0 +1,82 @@
+diff -Naur pexpect-4.1.0.orig/pexpect/async.py pexpect-4.1.0/pexpect/async.py
```

Comment:
It is. I thought about deleting the file in the build script, but this was easier for now.